test/commands/trigger_test.exs

```diff
@@ -1,13 +1,15 @@
 defmodule Cog.Test.Commands.TriggerTest do
-  use Cog.CommandCase, command_module: Cog.Commands.Trigger
+  use Cog.CommandCase, command_tag: :trigger
```

Comment:
@christophermaier we put them together to make the tests easier to run with tags. `mix test --only commands:trigger` vs `mix test --only commands:trigger-list --only commands:trigger-create ...`